awssqs/samples/README.md

```diff
@@ -11,21 +11,21 @@ commands from the root.
 
 ### Prerequisites
 
-1.  Create an [AWS SQS queue](https://aws.amazon.com/sqs/).
+1. Create an [AWS SQS queue](https://aws.amazon.com/sqs/).
 
-1.  Setup
-    [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+1. Setup
+   [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
 
-1.  The
-    [in-memory `ClusterChannelProvisioner`](https://knative.dev/eventing/tree/master/config/provisioners/in-memory-channel)
-    should be installed in your cluster. At the time of writing (release v0.5.0)
-    it is part of the default instructions so you will probably have it there
-    already.
+1. The
+   [in-memory `ClusterChannelProvisioner`](https://knative.dev/eventing/tree/master/config/provisioners/in-memory-channel)
```

Comment:
that thing is gone. CCP is removed, and is replaced by the CRD based channels

Reply:
Please update the yaml example files too.  Eg. w/ the new 0.9.0 release (coming on 18th Sep.), the `Subscription` is in the `messaging.knative.dev/v1alpha1` group

Also the ksvc uses older syntax, worth to update

Reply:
True, I see more occurences of `CCP` in the `eventing-contrib` repo. Are you ok with a separate PR for this? I already removed CCP from `docs` and `eventing` repo. Seems I forgot this one.

Reply:
Can do it immediately.

Reply:
@matzew

Reply:
We decided to change the context of the PR a bit and do the changes in one PR.